cypress/e2e/client-tasks/pagination.cy.js

```diff
@@ -11,17 +11,26 @@ describe("Pagination", () => {
           .invoke('val').should('contain', 'per-page=25')
     })
 
-    it("will not show previous arrow on page 1 but will show it on other pages", () => {
+    it("will not show previous on page 1 but will show it on other pages", () => {
       cy.get(".previous-page-pagination-link").should('not.exist')
-      cy.get("#top-nav .next-page-pagination-link").click()
+      cy.get('#top-nav > :nth-child(1) > .flex-container > .govuk-pagination__list > :nth-child(2) > .govuk-link').click()
```

Comment:
Addressing elements this way is really brittle. If the pagination component changes its implementation or styling, this test will fail, even though all it should be interested in is behaviour.

If you can't select the element by text, the next best option is to add `data-cy` attributes.

Reply:
Sorry I should've noticed this, just amended it based on the newer elements in the pagination